docs/info/privacy-policy.md

```diff
@@ -0,0 +1,369 @@
+Controller. We may use the services of various Service Providers in order to process your data more effectively.
+
+DATA SUBJECT is any living individual who is the subject of Personal Data.
+
+THE USER is the individual using our Service. The User corresponds to the Data Subject, who is the subject of Personal
+Data.
+
+## 3. Information Collection and Use
+
+We collect several different types of information for various purposes to provide and improve our Service to you.
+
+## 4. Types of Data Collected
+
+While using our Service, we may ask you to provide us with certain personally identifiable information that can be used
+to contact or identify you (“Personal Data”). Personally identifiable information may include, but is not limited to:
+
+0.1. Email address
+
+0.2. First name and last name
+
+0.3. Phone number
+
+0.4. Address, Country, State, Province, ZIP/Postal code, City
+
+0.5. Cookies and Usage Data
+
+We may use your Personal Data to contact you with newsletters, marketing or promotional materials and other information
+that may be of interest to you. You may opt out of receiving any, or all, of these communications from us by following
+the unsubscribe link.
+
+### Usage Data
+
+We may also collect information that your browser sends whenever you visit our Service or when you access Service by or
+through any device (“Usage Data”).
+
+This Usage Data may include information such as your computer’s Internet Protocol address (e.g. IP address), browser
+type, browser version, the pages of our Service that you visit, the time and date of your visit, the time spent on those
+pages, unique device identifiers and other diagnostic data.
+
+When you access Service with a device, this Usage Data may include information such as the type of device you use, your
+device unique ID, the IP address of your device, your device operating system, the type of Internet browser you use,
+unique device identifiers and other diagnostic data.
+
+### Tracking Cookies Data
+
+We use cookies and similar tracking technologies to track the activity on our Service and we hold certain information.
+
+Cookies are files with a small amount of data which may include an anonymous unique identifier. Cookies are sent to your
+browser from a website and stored on your device. Other tracking technologies are also used such as beacons, tags and
+scripts to collect and track information and to improve and analyze our Service.
+
+You can instruct your browser to refuse all cookies or to indicate when a cookie is being sent. However, if you do not
+accept cookies, you may not be able to use some portions of our Service.
+
+Examples of Cookies we use:
+
+0.1. **Session Cookies**: We use Session Cookies to operate our Service.
+
+0.2. **Preference Cookies**: We use Preference Cookies to remember your preferences and various settings.
+
+0.3. **Security Cookies**: We use Security Cookies for security purposes.
+
+### Other Data
```

Comment:
Are we collecting these?

Reply:
Github Pages might use cookies when defending against bots. Same applies to us if we device to use Cloudflare service in front of the ExpressLRS documentation.